examples/backend-adapters/server/src/convex.ts

```diff
@@ -0,0 +1,34 @@
+// In a real Convex app, you'd create this file at `convex/http.ts` to set
+// up HTTP routes for UploadThing.
+
+import { httpRouter } from "convex/server";
+
+import {
+  addUploadthingRoutes,
+  createUploadthing,
+  FileRouter,
+} from "uploadthing/convex";
+
+// In a real app, you can wire up `cereateUploadthing` to your Convex
+// schema for increased type safety.
+// ```ts
+// import schema from "./schema";
+// const f = createUploadthing({ schema })
+// ```
+const f = createUploadthing();
```

Comment:
It may be better to have a separate example showing the "real" way as you suggested in the PR comment. without it, this example doesnt really hold much value IMO

Reply:
sounds good, I'll remove this and port it to a full example.